src/app/api/github/repos/[owner]/[repo]/pulls/[pullNumber]/reviews/route.ts

```diff
@@ -55,13 +56,16 @@ export async function POST(
       })
       .then((response) => response.data)
       .catch((error: RequestError) => {
-        console.error('error', error)
-        throw createHttpError(error?.status)
+        throw createError(
+          error.name,
+          error.response.data.message,
+          error.response.data.status,
+          error.response.data.documentation_url
+        )
```

Comment:
Are you 100% sure `error.name`, `error.response.data.message`, `error.response.data.status`, and `error.response.data.documentation_url` are all valid data?